src/Parser/FrostMage/Modules/Items/ShardOfTheExodar.js

```diff
@@ -0,0 +1,44 @@
+import ITEMS from 'common/ITEMS';
+import SPELLS from 'common/SPELLS';
+import { formatNumber } from 'common/format';
+import Module from 'Parser/Core/Module';
+import Combatants from 'Parser/Core/Modules/Combatants';
+
+const BLOODLUST_BUFFS = [
+  SPELLS.HEROISM.id,
+  SPELLS.BLOODLUST.id,
+  SPELLS.TIME_WARP.id,
+  SPELLS.NETHERWINDS.id,
+  SPELLS.ANCIENT_HYSTERIA.id,
+  SPELLS.DRUMS_OF_FURY.id,
+  SPELLS.DRUMS_OF_RAGE.id,
+  SPELLS.DRUMS_OF_THE_MOUNTAIN.id,
+];
+
+class ShardOfTheExodar extends Module {
+
+  static dependencies = {
+		combatants: Combatants,
+	};
+
+	damage = 0;
+
+  on_initialized() {
+    this.active = this.combatants.selected.hasFinger(ITEMS.SHARD_OF_THE_EXODAR.id);
+  }
+
+  on_byPlayer_damage(event){
+    if (BLOODLUST_BUFFS.some(buff => this.combatants.selected.hasBuff(buff, event.timestamp))) {
+      this.damage += event.amount;
+    }
+  }
+
+  item() {
+    return {
+      item: ITEMS.SHARD_OF_THE_EXODAR,
+      result: `${this.owner.formatItemDamageDone(this.damage)} Total Damage During Lust`,
```

Comment:
'total damage during lust' is an almost completely useless metric to show the player for SotE. The purpose of the item bonus displays is showing the player how much extra they got from equipping the item. Try going for showing "extra lusts" or something. Might be a pain to calculate, but I'd prefer you left out this module entirely than use this stat.

Reply:
Yea, i tried to do the extra damage instead of total at one point, but all the examples of it that i found were examples where the legendary gave extra damage % not extra haste%, so i couldnt figure it out. And then i just left it at total. Probably should have just removed it. But i was considering changing it to (x possible lusts this fight) or something like that.